content/en/docs/v1/operations/services/velero-backup-configuration.md

```
@@ -66,8 +67,20 @@ spec:
     key: cloud
 ```
 
+`BUCKET_NAME` can be found with 
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)